Source/Relativity.Test.Helpers/Relativity.Test.Helpers.NUnit.Integration/WorkspaceHelpers/CreateWorkspaceHelperTests.cs

```diff
@@ -37,29 +28,25 @@ public void SetUp()
 		[OneTimeTearDown]
 		public void Teardown()
 		{
-			DeleteWorkspace.DeleteTestWorkspace(_workspaceId, _servicesManager, ConfigurationHelper.ADMIN_USERNAME,
-				ConfigurationHelper.DEFAULT_PASSWORD);
+			Helpers.WorkspaceHelpers.WorkspaceHelpers.Delete(_servicesManager, _workspaceId);
 			_testHelper = null;
 			_servicesManager = null;
 		}
 
 		[Test]
 		public void CreateWorkspaceTest()
-		{
-			_workspaceId = CreateWorkspace.CreateWorkspaceAsync(_workspaceName,
-				SharedTestHelpers.ConfigurationHelper.TEST_WORKSPACE_TEMPLATE_NAME, _servicesManager,
-				SharedTestHelpers.ConfigurationHelper.ADMIN_USERNAME, SharedTestHelpers.ConfigurationHelper.DEFAULT_PASSWORD).Result;
-
+		{

```

Comment:
Should use .ConfigureAwait(false)

Reply:
I changed this but it doesn't appear to be showing up haha